src/lib/Components/Bidding/Screens/BillingAddress.tsx

```diff
@@ -35,15 +35,19 @@ interface StyledInputInterface {
   }
 }
 
-const StyledInput = ({ label, error, onLayout, ...props }) => (
+interface StyledInputProps extends InputProps {
+  label: string
+  errorMessage?: string
+}
+const StyledInput: React.FC<StyledInputProps> = ({ label, errorMessage, onLayout, ...props }) => (
   <Flex mb={4} onLayout={onLayout}>
     <Serif size="3" mb={2}>
       {label}
     </Serif>
-    <Input mb={3} error={Boolean(error)} {...props} />
-    {!!error && (
+    <Input mb={3} error={Boolean(errorMessage)} {...props} />
+    {!!errorMessage && (
       <Sans size="2" color="red100">
-        {error}
+        {errorMessage}
```

Comment:
I had to rename `error` to `errorMessage` because `InputProps` already has an `error?: boolean` field that conflicted, and I wanted to extend that interface.